### ISSUE TITLE
Exception "Cannot read properties of undefined (reading 'ok')"

### ISSUE THREAD

**Issue by longzheng** [yenoiwesa/homebridge-daikin-airbase#49]
I've been trying to debug why sometimes this plugin gets stuck in a state where it doesn't refresh/poll for updates.

While debugging I noticed this in my Homebridge debug logs. I don't think this is the same issue, but it's interesting nevertheless.

```
C:\Users\Long\AppData\Roaming\npm\node_modules\homebridge-daikin-airbase\src\airbase-controller.js:129
        if (!response.ok) {
                      ^
TypeError: Cannot read properties of undefined (reading 'ok')
    at DaikinAircon.sendRequest (C:\Users\Long\AppData\Roaming\npm\node_modules\homebridge-daikin-airbase\src\airbase-controller.js:129:23)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at DaikinAircon.exec [as getControlInfo] (C:\Users\Long\AppData\Roaming\npm\node_modules\homebridge-daikin-airbase\src\utils.js:15:20)
    at DaikinAircon.updateSubscribedServices (C:\Users\Long\AppData\Roaming\npm\node_modules\homebridge-daikin-airbase\src\airbase-controller.js:213:39)
[10/10/2022, 8:59:46 pm] [Daikin Airbase] Child bridge process ended
[10/10/2022, 8:59:46 pm] [Daikin Airbase] Process Ended. Code: 1, Signal: null
```

The relevant code is this https://github.com/yenoiwesa/homebridge-daikin-airbase/blob/master/src/airbase-controller.js#L113-L131

I haven't fully debugged the issue but I believe if it fails 3 times because of a `FetchError`, then `response` will remain `undefined`, trying to execute `!response.ok` will throw.

I think one fix would be to check for `response` being `undefined` too

```
        if (!response) {
            throw `Request failed after ${RETRY_ATTEMPTS} attempts`;
        }
```

**Comment by yenoiwesa** [comment]
Hi @longzheng, thank you for the pull request!

I understand how the AirBase could not respond and end up with a `FetchError` (maybe the AirBase is busy and the request times out, or it purposely ignores it and does not respond), however I do not think this error will always be linked to three failed attempts so I will change the error thrown to something more generic.

**Comment by yenoiwesa** [comment]
No actually it's not even that, it's just that the request was never sent okay.

**Comment by yenoiwesa** [comment]
I published version `3.2.5` of the plugin on npm, Thank you for your contribution @longzheng!
